tests/modules/programs/neovim/plugin-config.nix

```diff
@@ -33,5 +41,13 @@ lib.mkIf config.test.enableBig {
       > "$vimout" || true
     assertFileContains "$vimout" "HM_EXTRA_CONFIG"
     assertFileContains "$vimout" "HM_PLUGINS_CONFIG"
+
+    initLua="$TESTED/home-files/.config/nvim/init.lua"
+    assertFileContains "$initLua" 'vim.cmd [[source '
+    assertFileContains "$initLua" 'function HM_PLUGIN_LUA_CONFIG ()'
+
+    flattened=$(mktemp)
+    sed ':a;N;$!ba;s/\n/__NL__/g' "$initLua" > "$flattened"
+    assertFileContains "$flattened" ']]__NL__function HM_PLUGIN_LUA_CONFIG ()__NL__end'
```

Comment:
thanks for adding a test. This feels a bit obscure. Might be easier comparing it to a snapshot file/golden test.

Reply:
I believe that would be `assertFileContent`

Reply:
yeah sry, I had assertFileContent in mind

Reply:
:white_check_mark: done